gh-133125: Clarify ZipInfo.date_time attribute documentation (#136082)
python/cpython: Doc/library/zipfile.rst

.. attribute:: ZipInfo.date_time

   The time and date of the last modification to the archive member.  This is a

>    tuple of six values representing the "last mod file time" and "last mod file date"

Reviewer: ```suggestion
   tuple of six values representing the "last [modified] file time" and "last [modified] file date"
```
I realize you are using the text from APPNOTE.txt here, but I think we should expand the word to be a bit clearer.

   fields from the ZIP file's central directory.

>    .. note::

Reviewer: I would move this note further down as "a tuple of six values" is lost as context when you get to "The tuple" below.

Reviewer: The note also seems very similar to the one below. I would merge the information in this note into the one below.

Author: I just added [5afd441](https://github.com/python/cpython/pull/136082/commits/5afd441892ea3d2b64f7c901da374cc2160c72c1) where I applied changes according to your opinion. Thank you

      The ZIP format supports multiple timestamp fields in different locations
      (central directory, extra fields for NTFS/UNIX systems, etc.). This attribute

>       specifically returns the timestamp from the central directory. Future Python
>       versions may use alternative timestamp sources if they provide better accuracy
>       or precision.

Reviewer: ```suggestion
      specifically returns the timestamp from the central directory.
```

I don't think we can change the source of `date_time` in a backwards compatible way.

   .. note::

      The ZIP format supports multiple timestamp fields in different locations
      (central directory, extra fields for NTFS/UNIX systems, etc.). This attribute
      specifically returns the timestamp from the central directory. The central
      directory timestamp format in ZIP files does not support timestamps before
      1980. While some extra field formats (such as UNIX timestamps) can represent
      earlier dates, this attribute only returns the central directory timestamp.

>       CPython interprets this timestamp as representing local time, not UTC.

Reviewer: ```suggestion
      The central directory timestamp is interpreted as representing local
      time, rather than UTC time, to match the behavior of other zip tools.
```
